pkg/cmd/synonyms/shared/flags_to_synonym.go

```diff
@@ -59,15 +59,21 @@ func FlagsToSynonym(flags SynonymFlags) (search.Synonym, error) {
 	}
 
 	// Default case
-	if flags.SynonymType == "" || flags.SynonymType == SynonymType(Regular) {
+	if flags.SynonymType == "" || flags.SynonymType == Regular {
```

Comment:
these error messages aren't used anymore since the interactive mode is now triggered when a flag is missing. 

I'm keeping them in case we want to have an option that disables interactive mode